api/func-e_run_test.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2025 Tetrate
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package api
+
+import (
+	"context"
+	"testing"
+
+	"github.com/tetratelabs/func-e/internal/test/e2e"
+)
+
+func TestRun(t *testing.T) {
```

Comment:
example of "e2e" run tests applied to the library mode, which works in a debugger